plugin/Record/RecordManagerReadForm.php

```diff
@@ -249,17 +271,19 @@ public function getNameByFormId($formId)
 	 *
 	 */
 
-	public function readFieldsByFormId($formId)
+	public function readFieldsByFormId(int $formId)
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlFields = "SELECT *
-				FROM {$prefix}oo_plugin_form_fieldconfig
-				WHERE `form_id` = ".esc_sql((int)$formId)."
-				ORDER BY `order` ASC";
+		$sql = $pWpDb->prepare(
+			"SELECT *
+			FROM `{$prefix}oo_plugin_form_fieldconfig`
+			WHERE `form_id` = %d",
+			$formId
+		);
```

Comment:
Hier fehlt auch das ORDER BY: 

```suggestion
		$sql = $pWpDb->prepare(
			"SELECT *
			FROM `{$prefix}oo_plugin_form_fieldconfig`
			WHERE `form_id` = %d ORDER BY `order` ASC",
			$formId
		);
```